android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1738,19 +1755,6 @@ public void setSrc(Source source) {
                     DataSourceUtil.getDefaultDataSourceFactory(this.themedReactContext, bandwidthMeter,
                             source.getHeaders());
 
-            // refresh custom Metadata
```

Comment:
@KrzysztofMoch I moved this part of code because it causes some crash in sample !